wifiphisher.py

```diff
@@ -362,7 +383,8 @@ def copy_AP():
     return channel, essid, mac
 
 def start_ap(mon_iface, channel, essid, args):
-    print '['+T+'*'+W+'] Starting the fake access point...'
+    print '['+T+'*'+W+'] Starting the fake access point on ' + mon_iface + '...'
+    time.sleep(2)
```

Comment:
So, I added the sleep here and mon_iface because the print statement already existed in the code but it was unusable because as soon as the print occurred the screen would clear so you never got to see the information this provided. Also, I added the var for the iface so that you could confirm which interface was being used for monitor mode.